.github/workflows/release-utils.yml

```
@@ -110,8 +110,23 @@ jobs:
+          message: ${{ steps.parse.outputs.error_message }}
+
+      - name: Clear Authorization Log
+        if: ${{ success() }}
+        uses: ./.github/actions/report-result
+        with:
+          alias: "Authorization"
+          cleanup: true
+
   sync-notes:
     name: Sync Release Notes
+    id: sync
```

actionlint (1.7.12)

[error] unexpected key "id" for "job" section. expected one of "concurrency", "container", "continue-on-error", "defaults", "env", "environment", "if", "name", "needs", "outputs", "permissions", "runs-on", "secrets", "services", "snapshot", "steps", "strategy", "timeout-minutes", "uses", "with"

(syntax-check)